storagemanager/users/views.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from .forms import UserRegisterForm, UserUpdateForm, ProfileUpdateForm
+from .models import Profile
+
+
+def register(request):
+    if request.method == 'POST':
+        form = UserRegisterForm(request.POST)
+        if form.is_valid():
+            # Create new user
+            form.save()
+
+            # Automatically log in new user
+            username = form.cleaned_data.get('username')
+            new_user = authenticate(username=username,
+                                    password=form.cleaned_data['password1'],
+                                    )
+            login(request, new_user)
+            messages.success(
+                request, f'{username}, your account has been created.')
+
+            # Add phone number to profile
+            user = User.objects.get(username=username)
+            profile = Profile.find_profile(user)
+            phone_number = form.cleaned_data.get('phone_number')
+            profile.add_phone_number(phone_number)
+
+            return redirect('profile')
+    else:
+        form = UserRegisterForm()
+    return render(request, 'users/register.html', {'form': form})
+
+
+@login_required
+def profile_update(request):
+    if request.method == 'POST':
+        u_form = UserUpdateForm(request.POST, instance=request.user)
```

Comment:
I'm not a big fun of such an abbreviations - I mean, here it is quite obvious but on the other side few more letters won't change much.